### PR TITLE
yaak 2024.6.6 (new cask)

### PR DESCRIPTION
**Pull request by chenrui333**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

```
yaak
  * Version '2024.6.6' differs from '' retrieved by livecheck.
```

### CODE REVIEW

File: Casks/y/yaak.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask "yaak" do
+  arch arm: "aarch64", intel: "x64"
+
+  version "2024.6.6"
+  sha256 arm:   "b91564f1b42587abc27a79b245b64367dcf0474fa5812a29c8ec39dd840e17ab",
+         intel: "d64ecd5245ed0eb7ad8f75182618b929644d7e4ad1590a24feb130f8affaeee6"
+
+  url "https://releases.yaak.app/releases/#{version}/yaak_#{version}_#{arch}.dmg"
+  name "yaak"
```
Comment (bevanjkay) [inline Casks/y/yaak.rb:9]:
```suggestion
  name "Yaak"

**Review by bevanjkay — COMMENTED** [review]
Also needs a `livecheck`

**Comment by chenrui333** [comment]
Did not quite anyway to check the latest version, might need some help on it.

**Comment by khipp** [comment]
I took a stab at a possible livecheck:

```ruby
  livecheck do
    url "https://yaak.app/download"
    regex(/href=.*?yaak[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
  end
```

**Comment by chenrui333** [comment]
> ```ruby
> https://yaak.app/download
> ```

actually that works for me, thanks! (I was thinking appcast LOL)

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**